### ISSUE TITLE
[1.x]SIP-Plugin Subscribe Header

### ISSUE THREAD

**Issue by doerner** [meetecho/janus-gateway#3077]
**What version of Janus is this happening on?** 
1.0.4

**Have you tested a more recent version of Janus too?**
no.

**Was this working before?**
no

**Is there a gdb or libasan trace of the issue?**
no

**Additional context**
Here is the problem:
If I subscribe to an sip-endpoint using the command 
		lines[0].sipcall.send({message: {request: 'subscribe', 
			event: 'dialog', 
			accept: 'application/dialog-info+xml', 
			subscribe_ttl: 3600, 
			call_id: callid , 
			to: 'sip:abc@domain.com'
		}});
everything works as expected on its first use.
Especially the first line "SUBSCRIBE sip:abc@domain.com SIP/2.0" is correctly generated using the value passed to the "to"-argument.
But on a subsequent call to a different target ("to") (and with a different callid of course), the "SUBSCRIBE" header does not use the "to"-argument of the new function call, but a contact value it seemed to have received from the OK-message from its previous subscribe answer. This results in an SIP-error ...

In short: The SIP-Header of a subsequent SIP-SUBSCRIBE is unexpected and unfortunately wrong.

**Comment by lminiero** [comment]
The SUBSCRIBE support in the plugin is currently limited. More precisely, it will not allow you to subscribe to different targets for the same thing (e.g., `dialog` in your case), so trying to do a new subscription for an event type that you already subscribed to will be interpreted as an update to the existing subscription. If multiple dialog subscriptions is indeed what you're trying to do, you'll need to spawn a helper session (associated with the SIP stack you created) and send an additional SUBSCRIBE of the same type on that handle instead, so that it doesn't overlap with the one that exists already.

**Comment by doerner** [comment]
Although this is not the answer I have hoped for. Thanks for your quick response.
